runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -2429,9 +2430,17 @@ void jitClassesRedefined(J9VMThread * currentThread, UDATA classCount, J9JITRede
          reportHookDetail(currentThread, "jitClassesRedefined", "  Notify CHTable on class old=%p fresh=%p", oldClass, freshClass);
          table->classGotRedefined(fe, oldClass, freshClass);
          }
+
       classPair = (J9JITRedefinedClass *) ((char *) classPair->methodList + (classPair->methodCount * sizeof(struct J9JITMethodEquivalence)));
       }
 
+#if defined(J9VM_OPT_JITSERVER)
+   if (compInfo->getPersistentInfo()->getRemoteCompilationMode() == JITServer::CLIENT)
+      {
+      compInfo->getUnloadedClassesTempList()->push_back(reinterpret_cast<TR_OpaqueClassBlock *>(~0));
```

Comment:
This may hard to understand for somebody not familiar with the code.
Let's add a comment and change that `~0` into a define or symbolic constant